azurerm/data_source_loadbalancer_backend_address_pool.go

```diff
@@ -0,0 +1,91 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+)
+
+func dataSourceArmLoadBalancerBackendAddressPool() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmLoadBalancerBackendAddressPoolRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+
+			"loadbalancer_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: azure.ValidateResourceID,
+			},
+
+			"backend_ip_configurations": {
+				Type:     schema.TypeSet,
```

Comment:
Should this be a list?

Reply:
(as above) removing this as it's more likely folks reference this for the ID